### ISSUE TITLE
Update model/view/display on pause

### ISSUE THREAD

**Issue by samreid** [phetsims/joist#458]
From https://github.com/phetsims/pendulum-lab/issues/187 it seems we need to add back code like 081275093e195f68aafec59773f9b45c94abc9e0 but that does not reschedule/stack animation frames.  @jonathanolson and @samreid can you please investigate?  We probably need to do this before all the LOL maintenance releases.

**Comment by samreid** [comment]
@jonathanolson can you please review/test?

**Comment by samreid** [comment]
I tried following the instructions in https://github.com/phetsims/pendulum-lab/issues/187 once but could not reproduce the problem even without the fix above.

**Comment by samreid** [comment]
Also please unhold https://github.com/phetsims/joist/issues/450 when complete

**Comment by jonathanolson** [comment]
> I tried following the instructions in phetsims/pendulum-lab#187 once but could not reproduce the problem even without the fix above.

I reverted the commit above (to the "broken" state) and was able to reproduce the bug on the first try.

I moved to master, but harness.legendsoflearning.com is now just not loading (tried on 3 devices, 6 browsers, and it looks like it's now infinite-looping on something).

I have a high confidence that the above commit should fix it though.

Will check back in a bit.
